artifactory/cli.go

```diff
@@ -533,15 +534,14 @@ func GetCommands() []cli.Command {
 			},
 		},
 		{
-			Name:            "curl",
-			Flags:           getCurlFlags(),
-			Aliases:         []string{"cl"},
-			Usage:           curldocs.Description,
-			HelpName:        common.CreateUsage("rt curl", curldocs.Description, curldocs.Usage),
-			UsageText:       curldocs.Arguments,
-			ArgsUsage:       common.CreateEnvVars(),
-			SkipFlagParsing: true,
```

Comment:
Yes, otherwise the --client-certificate-path et. al. would not be parsed.

Reply:
Okay this seems to be nontrivial. I had to remove it to support `--url` et. al. - before none of the cli options were supported for curl. This, however, breaks usage of actual curl options such as `-O`. Do you know a way to only parse known arguments (as in python)?

Reply:
The *curl* command supports only the *--server-id* flag, in terms of JFrog CLI native flags.
If we do not skip the flags parsing, the command will fail on bad usage.
The *--server-id* flag is extracted from the curl command in the following function:
```
// Returns Artifactory details
func (curlCmd *CurlCommand) GetArtifactoryDetails() (*config.ArtifactoryDetails, error) {
	// Get --server-id flag value from the command, and remove it.
	flagIndex, valueIndex, serverIdValue, err := utils.FindFlag("--server-id", curlCmd.arguments)
	if err != nil {
		return nil, err
	}
	utils.RemoveFlagFromCommand(&curlCmd.arguments, flagIndex, valueIndex)
	return config.GetArtifactorySpecificConfig(serverIdValue)
}
```

Reply:
Okay, I removed it again. It would be nice if we could support the flags required to build the artifactory details for curl as well. Maybe in another PR.